modules/nf-core/parabricks/haplotypecaller/main.nf

```diff
@@ -0,0 +1,60 @@
+process PARABRICKS_HAPLOTYPECALLER {
+    tag "$meta.id"
+    label 'process_high'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "Parabricks module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
```

Comment:
this needs to move into the script section. to make sure it is run at the right time in the pipeline: https://github.com/nf-core/modules/issues/3683